rai/node/lmdb.cpp

```diff
@@ -1770,18 +1770,17 @@ std::shared_ptr<rai::vote> rai::mdb_store::vote_get (rai::transaction const & tr
 std::vector<std::shared_ptr<rai::block>> rai::mdb_store::unchecked_get (rai::transaction const & transaction_a, rai::block_hash const & hash_a)
```

Comment:
An `unchecked_exists()` could also be helpful

Reply:
unchecked_exists for key hash? Or by unchecked_key (uint256+uint256)?
If by block own hash, then we need extra lookup table

Reply:
unchecked_key (successor/alternate and block hash)